src/functions/ducklake_add_data_files.cpp

```diff
@@ -291,7 +292,7 @@ FROM read_blob(%s)
 	}
 	// use parquet_file_metadata to get the num rows
 	result = transaction.Query(StringUtil::Format(R"(
-SELECT file_name, num_rows
+SELECT file_name, num_rows, footer_size
 FROM parquet_file_metadata(%s)
```

Comment:
`parquet_file_metadata` now also contains `file_size_bytes` - can we use that and get rid of the call to `read_blob` which is currently used to get the file size?